src/Admin/Vue.php

```diff
@@ -30,12 +30,9 @@ public function init() {
 			//TODO:: use the same get_option everywhere on the BO
 			$payplug_wooc_settings = get_option( 'woocommerce_payplug_settings', [] );
 
-			//TODO:: since we're making the call update all the values
-			if (!empty($payplug_wooc_settings["payplug_live_key"])) {
```

Comment:
you don't need to check permissions?
If not, do we need this flag/variable?

Reply:
the permissions are now checked when u click IP button and as the IP availability is still going to change i didnt make bigger changes, so i kept it here set to true just to keep showing the button instead of removing conditions that depend on this flag set to true